### DISCUSSION TITLE
Audio distortion on tablet players

### DISCUSSION THREAD

**Discussion by Kebo2r** [music-assistant/support#2615] [Q&A] [answered]
Gort MA up and running and really looking forward to the possibilities and utilities of this!  Have one issue I'm curious if there is a history with as this is still in development or if there is anything I can adjust on my end.  I am primarily wishing to play via two M10Plus 3rd Gen tablets.  I have them setup independently and as a group and play via Fully Kiosk.  The audio seems to distort while playing almost with a "throbbing" sound.  The audio plays but it almost sounds like it's "pulsing".  Unsure of a better description.  Maybe a sample rate issue or something?  Any ideas here?  Thank you!

**Comment by OzGav** [comment]
Possibly volume normalisation? Try turning that off and then report back.

**Reply by Kebo2r** [reply]
Turned that off with same result.  Thanks for the suggestion.

**Reply by OzGav** [reply]
Confirm you are using the native fully kiosk provider and not the HA provider

**Comment by Kebo2r** [comment]
Yes using Fully Kiosk and latest beta.Thanks Sent from my iPhoneOn Jul 12, 2024, at 18:44, OzGav ***@***.***> wrote:﻿
Confirm you are using the native fully kiosk provider and not the HA provider

**Reply by OzGav** [reply]
Have you tried with the option “enforce lossy mp3 stream” turned on?

**Comment by Kebo2r** [comment]
Yes.  Tried with and without.  Same result. :(

**Reply by OzGav** [reply]
Have you tried with different music providers?

**Comment by Kebo2r** [comment]
Yes even tried with local media on the HASS server.  Same symptom.Sent from my iPhoneOn Jul 12, 2024, at 19:18, OzGav ***@***.***> wrote:﻿
Have you tried with different music providers?

**Reply by OzGav** [reply]
Lastly then do you have another player you can try. You could use the Snapcast provider and send to a browser window.

**Reply by Kebo2r** [reply]
Ok just tried snap cast and it seems to resolve the issue.  Does that help narrow down?  Would love to keep things all inside Fully!  Thank you!

**Reply by OzGav** [reply]
Yes it would point to a fully kiosk issue. Will try and reproduce soon.

**Reply by Kebo2r** [reply]
@OzGav .  Have been doing some more testing and it appears the issue is not present on a low power ONN tablet using either Fully or Snapcast.  It appears it's related to the higher end Lenovo M10Plus 3rd gen tablet clients.  Any idea of on device settings or similar that I could check or that could cause an issue?  These ONN tablets are very slow so I don't think it's a power issue.  Thanks!

**Reply by OzGav** [reply]
I just tested my Tablet and it is working fine as well. The only settings available are what you have already found for the player. The last thing to test to see if it is a MA issue is add the tablet to HA via the Fully Kiosk Integration and then add the media player from there to MA via the HA Music Provider and see if the problem persists.

**Reply by OzGav** [reply]
Any updates?

**Reply by marcelveldt** [reply]
I remember a while back somebody with this tablet had some sort of feature enabled on the tablet that caused this, it had to do with calls. Sorry, thats all I got from my memory, hopefully it will get you to the solution.

**Comment by Kebo2r** [comment]
It seems to happen only when I put the players in a group.  Not even necessarily when playing to a group, just if the players exist in a group player.Sent from my iPhoneOn Jul 12, 2024, at 18:44, OzGav ***@***.***> wrote:﻿
Confirm you are using the native fully kiosk provider and not the HA provider

**Reply by OzGav** [reply]
Groups are being reworked at the moment so test again when the next beta comes out

**Reply by OzGav** [reply]
@Kebo2r beta 30 is released

**Reply by OzGav** [reply, marked as answer]
@Kebo2r hopefully the latest releases have helped you out

**Comment by Kebo2r** [comment]
@OzGav.  Thank you!  I’ll check it out and report back ASAP.  In the middle of a move right now so my smart home is in boxes at the moment.  :/Sent from my iPadOn Oct 27, 2024, at 04:54, OzGav ***@***.***> wrote:﻿
@Kebo2r hopefully the latest releases have helped you out
